docs/howto/serialize_poro.md

```diff
@@ -29,4 +29,10 @@ class MyModel < ActiveModelSerializers::Model
 end
 ```
 
-The default serializer would be `MyModelSerializer`.
\ No newline at end of file
+The default serializer would be `MyModelSerializer`:
+```ruby
+# my_model_serializer.rb
+class MyModelSerializer < ActiveModel::Serializer
+  attributes :id, :name, :level
+end
```

Comment:
@tommi-lew @bf4 I'm fine with any changes to this. I put that sentence there to point out the serializer is the same name as one created with ActiveRecord.